movement/watch_faces/complications/blinky_face.h

```diff
@@ -38,12 +38,12 @@ void blinky_face_activate(movement_settings_t *settings, void *context);
 bool blinky_face_loop(movement_event_t event, movement_settings_t *settings, void *context);
 void blinky_face_resign(movement_settings_t *settings, void *context);
 
-static const watch_face_t blinky_face = {
-    blinky_face_setup,
-    blinky_face_activate,
-    blinky_face_loop,
-    blinky_face_resign,
-    NULL
-};
+#define blinky_face ((const watch_face_t){ \
+    blinky_face_setup, \
+    blinky_face_activate, \
+    blinky_face_loop, \
+    blinky_face_resign, \
+    NULL, \
+})
```

Comment:
This was a tricky one, since the Emscripten compiler complained that the `static const` declarations _were not_ compile-time constants. (The ARM compiler was fine with either.) I had thought about alternatives for initializing the `watch_faces` array (like 8986710eacbf7bbf57f9938760a980dc38b16f3f) but I felt like this would be the _least weird_ to other developers.

Reply:
Definitely less weird 😄 and if it's well documented, it's as valid as any other approach. Eventually I want to write a script that automates all this anyway, so folks can just type something like `watch-face new tide_chart` and have all the boilerplate and makefile additions done for them.